django-backend/fecfiler/web_services/tasks.py

```diff
@@ -154,7 +150,7 @@ def submit_to_fec(
 
     """Submit to FEC"""
     try:
-        submission_type_key = DOT_FEC_KEY if not mock else MOCK_DOT_FEC_KEY
+        submission_type_key = EFO_SUBMITTER_KEY
```

Comment:
This breaks the functionality of being able to call the task and decide whether it will be mocked or not.  I get the goal of reducing how much code is different between the mock and the real submitter.  Maybe a way to do that without making the the submitter and print submitter tied to the environment variable is to just pull the logging out of the class so all that's left is the different part